examples/source/1.components/amp-script/index.html

```diff
@@ -158,6 +173,36 @@
         fetchCurrentTime();
       </script>
     </div>
+
+    <!--
+      ## Using a WebSocket for a live blog
+      `amp-script` supports WebSockets. This example simulates a live blog.
+    -->
+    <div>
+      <amp-script layout="fixed-height" height="200" script="live-blog-script" class="sample" sandbox="allow-forms">
```

Comment:
Does all the content fit into 200px?

Reply:
I used a scrolling container. I'd make it bigger, but actually this is enough space on the page to see that this works, and how it works.

Reply:
Not happy about this approach (but this is my general live blog concern). I'd preferred a different sample that better suits amp-script. But it's fine.

Reply:
I think it will inspire people to see that this sort of thing is possible. But I understand your concern!